content/docs/apps/managing-teammates.md

```diff
@@ -62,3 +62,95 @@ Decide which roles to give them. For a complete list of roles, see [the Cloud Fo
     cf set-org-role <email> <org> OrgAuditor
     ```
 {{% /commandline %}}
+
+## Removing a teammate
+
+If you are an Org Manager or Space Manager, you can remove teammates from orgs and spaces respectively.
+
+{{% dashboard %}}
+
+### Organization Users
```

Comment:
@rememberlenny I'll push a commit that lowercases `users` here and in the next headline — holler if you'd rather I not.